R/pkg/R/DataFrame.R

```diff
@@ -647,7 +645,7 @@ setMethod("toJSON",
             RDD(jrdd, serializedMode = "string")
           })
 
-#' write.json
+#' Save the contents of DataFrame as a JSON file
```

Comment:
Can we use SparkDataFrame as opposed to DataFrame (see https://issues.apache.org/jira/browse/SPARK-12148 for some more details).